app/jobs/technical_metadata_workflow_job.rb

```diff
@@ -38,6 +38,6 @@ def log_failure(druid:, errors:)
   end
 
   def client
-    @client ||= Dor::Workflow::Client.new(url: Settings.workflow.url)
+    @client ||= Dor::Workflow::Client.new(url: Settings.workflow.url, logger: Rails.logger)
```

Comment:
Note that the client allows you to pass in your own logger here: https://github.com/sul-dlss/dor-workflow-client/blob/main/lib/dor/workflow/client.rb#L36